### PR TITLE
Packaging performance-monitoring and node-monitoring

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Add files to build performance-monitoring
and node-monitoring package build

tendrl-bug-id: Tendrl/performance-monitoring#29
Signed-off-by: Timothy Asir J <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@anmolbabu, @r0h4n , @nthomas-redhat , @shtripat , Please review

File: node_monitoring/setup.py [hunk 1]
```diff
@@ -1,10 +1,107 @@
...
+def extract_requirements(filename):
+    with open(filename, 'r') as requirements_file:
+        return [
+            x[:-1] for x in requirements_file.readlines()
+            if not x.startswith("#") and x[:-1] != ''
+        ]
+
+install_requires = extract_requirements('requirements.txt')
+
+
```
Comment (shtripat) [inline node_monitoring/setup.py]: too many line gaps?

File: setup.py [hunk 2]
```diff
@@ -1,9 +1,115 @@
...
+            cmd.append('-s')
+        print(' '.join(cmd))
+        subprocess.check_call(cmd)
+
+        # Push Git tag to origin remote
+        cmd = ['git', 'push', 'origin', tag_name]
+        print(' '.join(cmd))
+        subprocess.check_call(cmd)
+
+        # Push package to pypi
```
Comment (shtripat) [inline setup.py]: Remove the commented lines if not required

File: node_monitoring/setup.py (hunk 1 above)
Comment (TimothyAsirJeyasing) [inline node_monitoring/setup.py]: pep8: expected 2 blank lines

File: setup.py (hunk 2 above)
Comment (TimothyAsirJeyasing) [inline setup.py]: Sure, I will remove the commented lines

**Comment by anmolbabu** [comment]
@TimothyAsir Please follow https://github.com/Tendrl/performance-monitoring/pull/34/files#diff-423eaed28c1d057218887cb6e5695734 for performance monitoring related installation steps and https://github.com/Tendrl/performance-monitoring/pull/34/files#diff-152b87d52bbca03020f803acc35ae115 for node_monitoring related installation steps. Also these steps will be valid once https://github.com/Tendrl/commons/pull/140 and https://github.com/Tendrl/performance-monitoring/pull/33 are merged

**Comment by r0h4n** [comment]
@TimothyAsir Please update this PR

**Comment by TimothyAsirJeyasing** [comment]
I want to split this patch into performance-monitoring and node-monitoring and fix all conflicts as per recent changes and send a separate patch.

**Pull request closed without merging**